a10_octavia/controller/worker/tasks/virtual_server_tasks.py

```diff
@@ -37,7 +37,12 @@ def set(self, set_method, loadbalancer):
         arp_disable = CONF.slb.arp_disable
         vrid = CONF.slb.default_virtual_server_vrid
         desc = loadbalancer.description
-        desc = "" if str(desc).isspace() else desc
+        if not desc:
+            desc = None
+        elif str(desc).isspace() or not str(desc):
```

Comment:
Is there any case for which `not desc`  and `not str(desc)`  results different value.

Reply:
Yes...
When we don't provide description parameter while creating lb, then `not desc` = **True**  and `not str(desc)` = **False**
Because at that time we get `desc` as empty but `str(desc)` as 'None'